angular-app/src/app/project-detail/project-detail.component.ts

```diff
@@ -16,8 +16,10 @@ import { ProjectService } from '../project.service';
 })
 export class ProjectDetailComponent implements OnInit {
 
+  private blobstoreUploadUrl: string;
   private files: string[] = ['audio.wav', 'audio.mp3', 'lyric1.txt', 'lyric2.docx']
```

Comment:
Is this test data needed any more if you can now upload blob with this pull?

Reply:
I thought I'd already deleted it . . . done now